templates/vue/src/components/modals/NodeModal/forms/ContentForm/AnswerForm.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div>
+    <b-form-group data-qa="activity-combobox" label="Select an activity:">
+      <combobox
+        v-model="activityNodeId"
+        :options="currentActivityNodes"
+        data-qa="choose-activity-node"
+        item-text="title"
+        item-value="id"
+        empty-message="There are no activities yet."
+      >
+        <template v-slot="slotProps">
+          <p>
+            {{ slotProps.option.title }}
+          </p>
+        </template>
+      </combobox>
+    </b-form-group>
+    <b-form-group
+      data-qa="question-select"
+      label="Select a question from that activity:"
+    >
+      <combobox
+        v-model="currentQuestion"
+        :options="getCurrentQuestions"
+        data-qa="choose-question"
+        item-text="text"
+        item-value="id"
+        empty-message="There is no activity selected yet."
+      >
+        <template v-slot="slotProps">
+          <p>
+            {{ slotProps.option.text }}
+          </p>
+        </template>
+      </combobox>
+    </b-form-group>
+    <b-form-group
+      label="Show this text first: "
+      description="If empty, will default to the follow-up text of the activity form."
+    >
+      <b-form-input v-model="followUpText" data-qa="follow-up-text"></b-form-input>
+    </b-form-group>
+  </div>
+</template>
+
+<script>
+import { mapState } from "vuex"
+import Combobox from "@/components/modals/common/Combobox"
+export default {
+  components: {
+    Combobox,
+  },
+  props: {
+    node: {
+      type: Object,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      activityNodeId: "",
+      currentQuestion: "",
+      followUpText: "",
+    }
+  },
+  computed: {
+    ...mapState(["nodes"]),
+    currentActivityNodes() {

```

Comment:
Since this is a computed property, it should be named like it's a property... i.e. `activityNodes`